src/HttpClientInterception/PublicAPI/net6.0/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1 @@
+#nullable enable
```

Comment:
The new bit _should_ go in the unshipped files until it actually gets to NuGet, then it'll get moved post-release.

However that's probably not worth doing if we just ship this on Monday with no other changes.